<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="LAB_Sample_Name">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;*Sample Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LE_Sample_Name"/>
   </item>
   <item>
    <widget class="QLabel" name="LAB_Sample_ID_2">
     <property name="text">
      <string>Sample ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LE_Sample_ID"/>
   </item>
   <item>
    <widget class="QLabel" name="LAB_Sample_Comment">
     <property name="text">
      <string>Sample Comment</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="TE_Sample_Comment"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
